### DISCUSSION TITLE
Badring Water Leak Sensor showing N/A

### DISCUSSION THREAD

**Discussion by juju1337** [Koenkk/zigbee2mqtt#23122] [Devices]
I have paired two IKEA Badring water leak sensors with zigbee2mqtt but it they only show the lqi, battery and water leak show "n/a" instead of "false/true".

**Comment by francisp2** [comment]
Take a wet cloth, put the Badring on it, and see it if changes.

**Reply by juju1337** [reply]
It didn't. I removed it and paired it again two times, now it works...
